stellarphot/tests/test_core.py

```diff
@@ -963,10 +963,36 @@ def test_catalog_from_vizier_search_apass():
 
 
 @pytest.mark.remote_data
-def test_catalog_from_vizier_search_vsx():
+@pytest.mark.parametrize("location_method", ["coord", "header", "wcs"])
+def test_catalog_from_vizier_search_vsx(location_method):
     # Do a cone search with a small enough radius to return exaclty one star,
     # DQ Psc, which happens to already be in the test data.
     coordinate = SkyCoord(ra=359.94371 * u.deg, dec=-0.2801 * u.deg)
+
```

Comment:
The logic of the test seems sound for testing your three location determining approaches.